README.md

```diff
@@ -29,11 +29,12 @@
 
 <!-- TODO nf-core: Include a figure that guides the user through the major workflow steps. Many nf-core
      workflows use the "tube map" design for that. See https://nf-co.re/docs/contributing/design_guidelines#examples for examples.   -->
-<!-- TODO nf-core: Fill in short bullet-pointed list of the default steps in the pipeline -->
 
 1. Subsample reads ([`Seqtk`](https://github.com/lh3/seqtk))
 2. Read QC ([`FastQC`](https://www.bioinformatics.babraham.ac.uk/projects/fastqc/))
-3. Present QC for raw reads ([`MultiQC`](http://multiqc.info/))
+3. Sequence statistics ([`SeqFu Stats`](https://github.com/telatin/seqfu2))
+4. Contamination QC ([`FastQ Screen`](https://www.bioinformatics.babraham.ac.uk/projects/fastq_screen/))
+5. Present QC for raw reads ([`MultiQC`](http://multiqc.info/))
```

Comment:
```suggestion
3. Report read counts and lengths statistics ([`SeqFu Stats`](https://github.com/telatin/seqfu2))
4. Screen reads against references ([`FastQ Screen`](https://www.bioinformatics.babraham.ac.uk/projects/fastq_screen/))
5. Compile QC results into report ([`MultiQC`](http://multiqc.info/))
```
Hi, Here are some suggestions for updated descriptions that aim to clarify further what each tool does and present them more succinctly. I also changed the MultiQC one as "raw reads" is not accurate when we now have subsampling. Let me know what you think! :)